packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -164,7 +164,38 @@ Media.createMediaConnection = (
       {
         iceServers,
       },
-      debugId
+      debugId,
+      (messages, context) => {
```

Comment:
this works fine, but I'm wondering if maybe this stuff should be moved into internal-media-core. SDK already passes a MediaLogger that implements a logging interface into internal-media-core - see the call to setLogger() in onReady() in meetings/index.ts. I feel it would be nicer to hide the details of how WCME logging works inside internal-media-core so SDK wouldn't need to know about it and it would use the same logging interface for RoapMediaConnection and MultistreamRoapMediaConnection

Reply:
Updated, moved this to `internal-media-core`